package.json

```diff
@@ -55,7 +55,7 @@
     "test:linkey": "node scripts/linkeySetup.js && jest src/app/lib/config/services/*.test.js --verbose true; yarn test:linkey:cleanup",
     "test:linkey:cleanup": "find src/app/lib/config/services -type f -name '*.test.js' -delete",
     "updateMinorPatch": "rm -rf node_modules/ && yarn install && npm update && yarn install",
-    "webpack:dev:client": "NODE_ENV=development webpack serve --inline --hot --env config='client'",
```

Comment:
From what I understand, there used to be two ways of doing an automatic refresh, an `iframe` mode which wraps the page in a refreshable iframe, and the `inline` mode, which adds the webpack-dev-server client entry point to the webpack config.

I think the `--inline` option has been removed as of v4 since webpack now take care of this for us, in the migration guide it specifies that there is no replacement for it either. Passing this option in at command line in v4 will stop the server from successfully spinning up so I think it has to go.